ci/checksizes.sh

```diff
@@ -34,20 +34,20 @@ REVERSI_OPT_GZ_SIZE=$(stat '-c%s' "$REVERSI_OPT.gz")
 
 case $FULLVER in
   2.11.12)
-    REVERSI_PREOPT_EXPECTEDSIZE=436000
-    REVERSI_OPT_EXPECTEDSIZE=94000
+    REVERSI_PREOPT_EXPECTEDSIZE=437000
+    REVERSI_OPT_EXPECTEDSIZE=95000
```

Comment:
If the optimizer removes the forwarders, why do we see a bump in code size?

Reply:
Oh that's because of the following situation. Let's say you have the following IR before the change:
```scala
module class Foo$ extends Object {
  // typically would not be inlined
  def bar(x: int, y: int): int = {
    val z = x * y
    x + y + z
  }
}

def main() {
  mod:Foo$.bar(someInt(), someOtherInt())
}
```
Since the call to `mod:Foo$.bar` is not inlined, its arguments stay as is, their expressions inline in the parameters at the JS level:
```js
main__AT__V() {
  $m_Foo$().bar(someInt(), someOtherInt())
}
```
However, now if we have the IR with the static methods:
```scala
class Foo extends Object {
  static def bar(x: int, y: int): int = {
    mod:Foo$.bar(x, y)
  }
}

def main() {
  Foo::bar(someInt(), someOtherInt())
}
```
The call to `Foo::bar` is now inlined, so its non-pure arguments must be assigned to temporary `val`s. Since `someInt()` and `someOtherInt()` are not known to be pure, the JS code for main ends up being:
```js
main__AT__V() {
  const x$1 = someInt();
  const x$2 = someOtherInt();
  $m_Foo$().bar(x$1, x$2)
}
```
which is longer in terms of character count.

This situation already happens before this PR for all sorts of "fowarder" methods. We could one day improve this in the optimizer by identifying that forwarder methods can be inlined by directly splicing their actual arguments in place of the formal parameters, if all the formal parameters are used exactly once, in the same evaluation order, before any non-pure operation is performed by the body of the method. In particular, that would be easy to prove for any static forwarder if the module has a pure constructor (which is an analysis we already perform).